### PR TITLE
Renamed converter builders to lowerCase.

### PR DESCRIPTION
**Pull request by oskarwrobel**
Fixes: #467 

Related to:
- https://github.com/ckeditor/ckeditor5-basic-styles/pull/10
- https://github.com/ckeditor/ckeditor5-typing/pull/22
- https://github.com/ckeditor/ckeditor5-paragraph/pull/5
- https://github.com/ckeditor/ckeditor5-headings/pull/13

### CODE REVIEW

**Comment by scofalik** [comment]
You haven't touched the file name, what was suggested in the issue. However, I don't like file name `buildmodelconverterfor.js`. I think that we could do some minor refactor here:
* name the function `buildModelConverter`
* in `ModelConverterBuilder`, remove initializing the class with dispatchers (same for `buildModelConverter`)
* add `for()` method that will take dispatchers as a parameter
* change how the converter builders are used (this should be easy search+replace)

So now it will look like this `buildModelConverter.for( dispatchers ).fromXXX( ... ).toXXX( ... );`. In my opinion, it looks better and we will have a better looking function/file name (`buildmodelconverter.js`).

Same applies to view converters builder.

**Comment by Reinmar** [comment]
Doh... I've just spent 15 minutes reviewing the changes :D. I haven't noticed it's R-ed already. Anyway, I think that I agree with changes proposed by @scofalik.

**Comment by Reinmar** [comment]
PS. This PR could have a branch in ckeditor5 with configuration of the branches. It took me some time to switch all that.

**Comment by oskarwrobel** [comment]
@scofalik If I understood the idea correctly `buildModelConverter` will be a function, so it will work like this:
```javascript
buildModelConverter().for( dispatchers ).fromXXX( ... ).toXXX( ... );
```
instead of
```javascript
buildModelConverter.for( dispatchers ).fromXXX( ... ).toXXX( ... );
```

**Comment by pjasiun** [comment]
But then we will still have different file name and function name don't we?

What do you think about skipping "build", so we will have:
```
modelConverter.for( dispatchers ).fromXXX( ... ).toXXX( ... );
```
In the `ModelConverter` file?

**Comment by scofalik** [comment]
As far as second version looks and reads better, I am not sure if there is an elegant solution that would enable it. Anyway, both are fine with me.

**Comment by oskarwrobel** [comment]
Maybe something like this:
```javascript
export default class ModelConverter {
    ...
    static for ( ...dispatchers ) {
        return new this( dispatchers );
    }
    ...
}
```

and then

```javascript
import ModelConverter from 'path/to/modelconverter.js';

ModelConverter.for()...
```

**Comment by pjasiun** [comment]
> @pjasiun: It's fine and it was one of ideas we discussed earlier with @oskarwrobel . But it does not denote action, so the whole "sentence" has less sense. There is a feeling that it returns something that you have to use afterwards.

I get your point and it makes sens for me.

> Maybe...

Now all proposed options are fine for me, as long the file name is the same as the function/class name.

File: src/conversion/buildmodelconverter.js [hunk 1]
```diff
@@ -87,6 +84,18 @@ class ModelConverterBuilder {
 	}
 
 	/**
+	 * Set one or more dispatchers which the built converter will be attached to.
+	 *
+	 * @chainable
+	 * @param {...engine.conversion.ModelConversionDispatcher} dispatchers One or more dispatchers.
+	 */
```
Comment (oskarwrobel) [inline src/conversion/buildmodelconverter.js]: Missing `@returns {engine.conversion.ModelConverterBuilder}`

File: src/conversion/buildviewconverter.js [hunk 2]
```diff
@@ -103,9 +100,21 @@ class ViewConverterBuilder {
 	}
 
 	/**
+	 * Set one or more dispatchers which the built converter will be attached to.
+	 *
+	 * @chainable
+	 * @param {...engine.conversion.ViewConversionDispatcher} dispatchers One or more dispatchers.
```
Comment (oskarwrobel) [inline src/conversion/buildviewconverter.js]: Missing `@returns {engine.conversion.ViewConverterBuilder}`

**Pull request merged**